### PR TITLE
php@5.6: import from homebrew/php

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Preliminary PR, it doesn't even build yet

### CODE REVIEW

**Comment by ilovezfs** [comment]
@kabel @javian any thoughts here?

File: Formula/php@5.6.rb [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+      --with-pdo-dblib=#{Formula["freetds"].opt_prefix}
+      --with-pdo-mysql=mysqlnd
+      --with-pdo-odbc=unixODBC,#{Formula["unixodbc"].opt_prefix}
+      --with-pdo-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-pic
+      --with-png-dir=#{Formula["libpng"].opt_prefix}
+      --with-pspell=#{Formula["aspell"].opt_prefix}
+      --with-tidy
```
Comment (kabel) [inline Formula/php@5.6.rb]: Remove tidy

**Review by kabel — CHANGES_REQUESTED** [review]
Should we also add mcrypt here? It's also pretty high on the analytics.

**Comment by ilovezfs** [comment]
>Should we also add mcrypt here? It's also pretty high on the analytics.

yup

File: Formula/php@5.6.rb [hunk 2]
```diff
@@ -0,0 +1,380 @@
+class PhpAT56 < Formula
+  desc "General-purpose scripting language"
+  homepage "https://secure.php.net/"
+  url "https://php.net/get/php-5.6.34.tar.xz/from/this/mirror"
```
Comment (kabel) [inline Formula/php@5.6.rb]: 5.6.35 out now

File: Formula/php@5.6.rb [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+      --with-jpeg-dir=#{Formula["jpeg"].opt_prefix}
+      --with-kerberos
+      --with-layout=GNU
+      --with-ldap
+      --with-ldap-sasl
+      --with-libedit
+      --with-libzip
+      --with-mhash
+      --with-mysql-sock=/tmp/mysql.sock
+      --with-mysqli=mysqlnd
```
Comment (javian) [inline Formula/php@5.6.rb]: Can you also add `--with-mysql=mysqlnd` to enable the old mysql extension ?

File: formula_renames.json [hunk 4]
```diff
@@ -136,6 +136,7 @@
   "perl518": "perl@5.18",
   "php71": "php@7.1",
   "php72": "php",
+  "php56": "php@5.6",
```
Comment (ilovezfs) [inline formula_renames.json]: should be in alphabetical order

**Review by kabel — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks again @SMillerDev! 🚀

**Pull request closed without merging**